notes.md

```diff
@@ -226,6 +228,16 @@
     2019.4.2f1
     2019.4.3f1
     2019.4.4f1
+    2019.4.5f1
+    2019.4.6f1
+    2019.4.7f1
+    2019.4.8f1
+    2019.4.9f1
+    2020.1.0f1 *
```

Comment:
Can you detail the structure changes corresponding to this `*` in the final section of `notes.md`? (I know things might not be final yet - but you can always update the description if things change).

Reply:
The 2020.1.0f1 changes are final, it's the 2020.2 stuff that is in the air. I have not committed anything related to that. Do you want me to push PRs for alphas and betas?

Reply:
Notes.md updated